templates/ovndbcluster/bin/functions

```diff
@@ -18,3 +18,15 @@ DB_FILE=/etc/ovn/ovn${DB_TYPE}_db.db
 function cleanup_db_file() {
     rm -f $DB_FILE
 }
+
+function wait_for_ovsdb_tool {
+    while true; do
```

Comment:
actually this could probably just smth like

```
while pgrep -a ovsdb-tool; do
  echo ...
  sleep 1
done
```

Reply:
Done